NabtoEdgeClientHello/Podfile

```diff
@@ -2,5 +2,14 @@ source 'https://github.com/CocoaPods/Specs.git'
 
 target 'NabtoEdgeClientHello' do
     use_frameworks!
-    pod 'NabtoEdgeClientSwift'
+    pod 'NabtoEdgeClientSwift', '2.2.0-beta.4'
```

Comment:
the dependency is from a the Hello world app that comes as part of the bundle .. but it should still be a proper release. It will be updated when the release has been made